gpio/__init__.py

```diff
@@ -16,7 +17,9 @@
 GPIO_ROOT = '/sys/class/gpio'
 GPIO_EXPORT = os.path.join(GPIO_ROOT, 'export')
 GPIO_UNEXPORT = os.path.join(GPIO_ROOT, 'unexport')
-FMODE = 'w+'  # w+ overwrites and truncates existing files
+FMODE_SYS_WO = 'w'   # /sys/class/gpio/export is not readable, even by root

```

Comment:
Why SYS and not STR?

Reply:
No good reason. I was focused on the permissions issues which are somewhat unique to /sys/ (sysfs). But since all of your opens are to sysfs, SYS is not a particularly good differentiator. STR sounds very reasonable to me.